src/print.jl

```diff
@@ -819,7 +819,7 @@ function con_str(mode, c::GenericRangeConstraint, sym)
     out_str
 end
 # Backwards compatability shim
-conToStr(c::GenericRangeConstraint) = con_str(REPLMode,c)
+constr(c::GenericRangeConstraint) = con_str(REPLMode,c)
```

Comment:
maybe `conToStr` -> `constraintstr`
These "shims" are definitely from me, was some lazyness during the upgrading of printing iirc